packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -525,7 +487,7 @@ export function getValidatorApi(
           graffiti ?? getDefaultGraffiti(getLodestarClientVersion(opts), chain.executionEngine.clientVersion, opts)
         ),
         feeRecipient,
-        commonBlockBody,
+        commonBlockBody: await commonBlockBodyPromise,
```

Comment:
if we await it here, doesn't it wait for common block body to be produced before we sent request to execution layer?

Reply:
+1